src/elmExecutor/buildPopulatedResourceBundle.js

```diff
@@ -1,5 +1,35 @@
-function doSearch(smart, type, callback) {
+function doSearch(smart, type, fhirVersion, request, callback) {
   const q = {};
+  
+  // setup the query for Practitioner and Coverage
+  switch (type) {
+    case "Practitioner":
+      q._id = request.performer.reference;
+      console.log(q._id);
+      break;
+    case "Coverage":
+      switch (fhirVersion.toUpperCase()) {
+        case "STU3":
+          if (request.extension.length > 0) {
```

Comment:
same thing here, are we certain `request` has an `extension` and won't return null?